cmake/configs/qurt_eagle_legacy_driver_default.cmake

```diff
@@ -63,10 +63,17 @@ set(config_module_list
 	# PX4 drivers
 	#
 	drivers/gps
-	drivers/uart_esc
+	#drivers/uart_esc
```

Comment:
It looks like the `uart_esc` module conflicts with the current implementation and one of the two needs to be renamed.